src/main/java/codeu/model/store/basic/MessageStore.java

```diff
@@ -85,6 +85,20 @@ public void addMessage(Message message) {
     return messagesInConversation;
   }
 
+  /** Find and return the Message with the given UUID. */
+  public Message getMessageWithId(UUID id) {
```

Comment:
Is `getMessageWithId` used in this PR anywhere? Is it necessary?

Reply:
If you think it will be useful for a future change, I'd ask you to split it into a separate PR. If you're not sure you'll need it, YAGNI principle should apply 😄

Reply:
`getMessageWithId` is never used, however `getConversationWithId` and `getUserWithId` both are.